server/modules/getAccessAndRefreshTokens.js

```diff
@@ -1,12 +1,38 @@
 const querystring = require("querystring");
+const { Pool } = require("pg");
 
 const getAccessAndRefreshTokens = async function (req, res) {
   const TOKEN_URL = "https://accounts.spotify.com/api/token";
-  const REDIRECT_URI = "http://localhost:3001/callback";
+  const REDIRECT_URI = process.env.REDIRECT_URI;
   const CLIENT_ID = process.env.CLIENT_ID;
   const CLIENT_SECRET = process.env.CLIENT_SECRET;
   const code = req.query.code || null;
 
+  // db connection
+  const pool = new Pool({
+    user: process.env.DB_USER,
+    host: process.env.DB_HOST,
+    database: process.env.DB_NAME,
+    password: process.env.DB_PASSWORD,
+    port: process.env.DB_PORT,
+    ssl: true,
+  });
+
+  // store the refresh token in DB instead of console logging it
+  const storeRefreshTokenInDB = async function (refresh_token) {
+    pool.query(
+      "INSERT INTO refresh_tokens_table (id, refresh_token) VALUES (1, $1) ON CONFLICT (id) DO UPDATE SET refresh_token = $1",
```

Comment:
Question: I was thinking about this part of the code from Saturday, and I wondered if an UPDATE statement was tried? 

![image](https://github.com/m4ndycheung/cyf-stay-tuned-radio/assets/109825538/954cef5c-5c1e-49b6-9a8d-3da5744c2ac8)